test/e2e/pivoting.go

```diff
@@ -479,26 +395,21 @@ func rePivoting(ctx context.Context, inputGetter func() RePivotingInput) {
 	})
 
 	By("Reinstate BMO in Source cluster")
-	installIronicBMO(ctx, func() installIronicBMOInput {
-		return installIronicBMOInput{
-			ManagementCluster:          input.BootstrapClusterProxy,
-			BMOPath:                    input.E2EConfig.GetVariable(bmoPath),
-			deployIronic:               false,
-			deployBMO:                  true,
-			deployIronicTLSSetup:       getBool(input.E2EConfig.GetVariable(ironicTLSSetup)),
-			deployIronicBasicAuth:      getBool(input.E2EConfig.GetVariable(ironicBasicAuth)),
-			deployIronicKeepalived:     getBool(input.E2EConfig.GetVariable(ironicKeepalived)),
-			deployIronicMariadb:        getBool(input.E2EConfig.GetVariable(ironicMariadb)),
-			Namespace:                  input.E2EConfig.GetVariable(ironicNamespace),
-			NamePrefix:                 input.E2EConfig.GetVariable(NamePrefix),
-			RestartContainerCertUpdate: getBool(input.E2EConfig.GetVariable(restartContainerCertUpdate)),
-			E2EConfig:                  input.E2EConfig,
-			SpecName:                   input.SpecName,
-		}
+	bmoKustomization := input.E2EConfig.GetVariable("BMO_RELEASE_LATEST")
+	By(fmt.Sprintf("Installing BMO from kustomization %s on the upgrade cluster", bmoKustomization))
```

Comment:
"Upgrade cluster" is confusing to me. We call it bootstrap or source cluster in other places so I would suggest changing to that.
```suggestion
	By(fmt.Sprintf("Installing BMO from kustomization %s on the bootstrap cluster", bmoKustomization))
```